SensorMap.h

```diff
@@ -7,28 +7,31 @@
 #define SENSORMAP_H
 
 #include "arduino.h"
-#include <NewPing.h>
-#include <i2cmaster.h>
+#include "Wire.h"
+#include <Adafruit_TCS34725.h>
+#include <Ultrasonic.h>
+#include "Multiplexor.h"
 
-#define SONAR_NUM 6      // Number of sensors.
-#define kSonarMaxDistance 200 // Maximum distance (in cm) to ping.
+#define TCAADDR 0x70          // Multiplexor numbers.
```

Comment:
You're defining this on two different files, if you're gonna use it on several files put it on the common.